src/test/java/domain/NameTest.java

```diff
@@ -0,0 +1,24 @@
+package domain;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+
+class NameTest {
+
+    @DisplayName("자동차 이름은 1자 이상, 5자 이하여야 한다.")
+    @Test
+    void test() {
+        assertDoesNotThrow(() -> new Name("쥬"));
+        assertDoesNotThrow(() -> new Name("쥬니"));
+        assertDoesNotThrow(() -> new Name("쥬니헙크1"));
+
+
+        assertThatThrownBy(() -> new Name(""))
+                .isInstanceOf(IllegalArgumentException.class);
+        assertThatThrownBy(() -> new Name("123456"))
+                .isInstanceOf(IllegalArgumentException.class);
```

Comment:
테스트는 성공케이스와 실패케이스를 분리하여 짜는 게 가독성이 좋습니다만, 굳이 한 메소드로 하고 싶다면 CsvSource를 고려한 ParameterizedTest를 적용해도 좋습니다. 정답은 없으니 원하시는 쪽으로 리팩토링해보아요 🙂

Reply:
```suggestion
    @DisplayName("1자 이상, 5자 이하로 생성할 수 있다.")
    @ParameterizedTest
    @ValueSource(strings = {"쥬", "쥬니", "쥬니헙크1"})
    void test1(String name) {
        assertDoesNotThrow(() -> new Name(name));
    }

    @DisplayName("1자 이상, 5자 이하가 아닌 경우, 생성할 수 없다.")
    @ParameterizedTest
    @ValueSource(strings = {"", "123456"})
    void test2(String name) {
        assertThatThrownBy(() -> new Name(name))
                .isInstanceOf(IllegalArgumentException.class);
    }
```
조언해주신 것처럼, 가독성을 위해 성공 테스트와 실패 테스트를 분리하였습니다.